### ISSUE TITLE
Not able to use repl when using external packages

### ISSUE THREAD

**Issue by NickSeagull** [theam/haskell-do#30]
I started working on a Kaggle problem with HaskellDO, but I wasn't able to get my code to compile (it is attached).

I've also attached the cabal and stack YAML files.

# Lib.hs

```haskell
-- # 1. Introduction
-- This is a direct port of the [Exploring survival on the titanic notebook](https://www.kaggle.com/mrisdal/titanic/exploring-survival-on-the-titanic) by Megan Risdal.
-- The intention of this is to have an example of what can be achieved with the data science tools we have in Haskell as for **February 27th, 2017**.

{-# LANGUAGE ConstraintKinds #-}
{-# LANGUAGE DataKinds #-}
{-# LANGUAGE FlexibleContexts #-}
{-# LANGUAGE GADTs #-}
{-# LANGUAGE OverloadedStrings #-}
{-# LANGUAGE PatternSynonyms #-}
{-# LANGUAGE QuasiQuotes #-}
{-# LANGUAGE ScopedTypeVariables #-}
{-# LANGUAGE TemplateHaskell #-}
{-# LANGUAGE TypeOperators #-}
{-# LANGUAGE ViewPatterns #-}
module Lib where
             
import Control.Applicative
import qualified Control.Foldl as L
import qualified Data.Foldable as F
import Data.Proxy (Proxy(..))
import Lens.Family
import Frames
import Frames.CSV (readTableOpt, rowGen, RowGen(..))
import Pipes hiding (Proxy)
import qualified Pipes.Prelude as P
-- ## 1.1 Load and check data
-- We begin by loading our data from the `input/train.csv` file. `Frames` generates a data type in *compile time* for our CSV file.

tableTypes "Train" "input/train.csv"
```

# Cabal file

```text
name:                ex01-exploring-titanic
version:             0.1.0.0
-- synopsis:
-- description:
homepage:            https://github.com/githubuser/ex01-exploring-titanic#readme
license:             BSD3
license-file:        LICENSE
author:              Author name here
maintainer:          example@example.com
copyright:           2017 Author name here
category:            Web
build-type:          Simple
extra-source-files:  README.md
cabal-version:       >=1.10

library
  hs-source-dirs:      src
  build-depends:       base >= 4.7 && < 5
                     , Chart-diagrams
                     , Frames
                     , lens
                     , lens-family-core
                     , pipes
                     , foldl
  default-language:    Haskell2010

source-repository head
  type:     git
  location: https://github.com/githubuser/ex01-exploring-titanic
```

# Stack.yaml

```yaml
resolver: lts-8.3
packages:
- '.'
extra-deps: []

flags: {}

extra-package-dbs: []
```

**Comment by kitfre** [comment]
So I've modified HaskellDO to output error messages into the console and identified the error. Here's a paste of running HaskellDO on the input files you provided.

```
Lib.hs:1:1: error:
    Exception when trying to run compile-time code:
      input/train.csv: openFile: does not exist (No such file or directory)
    Code: tableTypes "Train" "input/train.csv"

<no location info>: error:
    Could not find module `Lib'
    It is a member of the hidden package `kaggle-0.1.0.0@kaggle-0.1.0.0-JrgEDvSSqZh6hUmr8uJNPn'.

C:\Users\kitfr\code\temp\kaggle\src\Lib.hs:1:1: error:
    Exception when trying to run compile-time code:
      input/train.csv: openFile: does not exist (No such file or directory)
    Code: tableTypes "Train" "input/train.csv"
```
This should be a quick fix, it's just a matter of ensuring that HaskellDO spawns in the right place, I'm on it! 👍 

Edit: For clarification, the bug here is not the repl not working with external libraries, but that we simply spawned the repl in the wrong folder, I'm working on fixing that right now.

**Comment by kitfre** [comment]
Closed with #31, and #29 as well?

**Comment by NickSeagull** [comment]
Fixed!
